sys/kern/exec_elf.c

```diff
@@ -120,7 +120,16 @@ exec_elf_check(struct exec_params *epp)
     }
     if (epp->hdr.elf.e_type != ET_EXEC)
         return ENOEXEC;
-    if (epp->hdr.elf.e_machine != EM_MIPS || epp->hdr.elf.e_version != EV_CURRENT)
+    #if defined(__mips__)
```

Comment:
Could be `#ifdef __mips__` to be consistent with the rest of `/sys/kern`.
Don't indent macros.